cmd/livegrep-github-reindex/main.go

```diff
@@ -140,17 +141,14 @@ func main() {
 	if err := writeConfig(config, configPath); err != nil {
 		log.Fatalln(err.Error())
 	}
-	if *flagNoIndex {
-		log.Printf("Skipping indexing after writing config")
-		return
-	}
 
 	index := flagIndexPath.Get().(string)
 
 	args := []string{
 		"--out", index,
 		"--codesearch", *flagCodesearch,
 		"--num-workers", *flagNumRepoUpdateWorkers,
+		"--no-index", strconv.FormatBool(*flagNoIndex),
```

Comment:
I think you'll need to conditionally append this instead like the other bool flags below or use `--no-index=...` to make it one arg. A lot of CLI parsers ignore separate bool flag args and treat the `false` as an arg, and commands that don't handle args often just drop them on the floor instead of asserting they're empty.